dags/openshift_nightlies/config/benchmarks/hosted-control-plane.json

```diff
@@ -16,7 +21,9 @@
                 "JOB_TIMEOUT": "18000",
                 "QPS": "20",
                 "BURST": "20",
-                "STEP_SIZE": "30s",
+                "STEP_SIZE": "60s",
+                "HYPERSHIFT": "true",
```

Comment:
where is this env var being used?

Reply:
@Harshith-umesh `HYPERSHIFT` is used in e2e script [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/common.sh#L40) for conditional tasks and `STEP_SIZE` is used in kube-burner ripsaw yaml [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/kube-burner-crd.yaml#L45) for prometheus step size.

Reply:
interesting.